src/components/Table/Table.tsx

```diff
@@ -279,7 +280,8 @@ function Table<Values extends Record<string, unknown>>({
   hasCustomDateFormat,
   customDateFormat,
   hasResizableColumns = false,
-  hasEmptyActionsCell = false
+  hasEmptyActionsCell = false,
+  isWH = false
```

Comment:
I'm not sure is the right name for the param
i guess it's better to have meaningful name